src/engine/utils/create-patch.ts

```diff
@@ -1,17 +1,18 @@
 import { Entry } from 'contentful'
-import { create as createDiffer, Delta, formatters as diffFormatters, Patch } from '@contentful/jsondiffpatch'
-
-const format: (delta: Delta | undefined) => Patch = diffFormatters.jsonpatch.format
-
-const entryDiff = createDiffer({
-  propertyFilter: function (name: string) {
-    return !['sys'].includes(name)
-  },
-  textDiff: {
-    minLength: Number.MAX_SAFE_INTEGER,
-  },
-})
+import { generateJSONPatch, pathInfo } from 'generate-json-patch'
 
 export const createPatch = ({ targetEntry, sourceEntry }: { targetEntry: Entry<any>; sourceEntry: Entry<any> }) => {
-  return format(entryDiff.diff(targetEntry, sourceEntry))
+  // Temp fix until the types from generate-json-patch are fixed
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  return generateJSONPatch(targetEntry, sourceEntry, {
+    propertyFilter: function (name: string, context) {
+      const { length } = pathInfo(context.path)
+      // ignore sys prop on root level
+      if (length === 1) {
```

Comment:
should we also ignore metadata here?

Reply:
@ruderngespra good point. 

Ignoreing it now - also added a test for it!

Reply:
I'd personally advise against it. We already exclude metadata at an early step in the pipeline so it should never get this far. I am not sure if creating more places where we ignore it makes it easier to understand and change it a later point.

Reply:
That's a valid point @andreascful - please feel free to revert it with another PR as this is already merged 🙏